test/unit/tasks/emergency_banner_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+require_relative '../../../lib/emergency_banner/deploy'
+require_relative '../../../lib/emergency_banner/remove'
+
+describe 'emergency_banner:deploy' do
```

Comment:
is there any benefit to keeping these here rather than in `test/unit/emergency_banner/deploy_test.rb` etc?